### PR TITLE
feat: skip installation of attic when installed

### LINKED ISSUE #20: Proposal: Skip installation when `attic` in `PATH`
Hello,
I'm currently trying to use this action on a "host"-runner. 
Sadly the installation of `attic` using the script doesn't work, but I can add `attic` to the PATH using the provided flake.

It would be nice if the action would skip the attic installation if the binary is already present.

What do you think? I would be happy to implement the feature if you are ok with it.

### PR DESCRIPTION
**Pull request by pfzetto**
This PR adds a check to skip the installation of `attic` when `attic -V` returns with exit code 0.

Closes #20

### CODE REVIEW

File: src/stages/install.ts [hunk 1]
```diff
@@ -33,3 +33,8 @@ export const install = async () => {
 
 	core.endGroup();
 };
+
+export const is_installed = async () => {
```
Comment (ryanccn) [inline src/stages/install.ts:37]:
```suggestion
export const isInstalled = async () => {
```

File: src/stages/install.ts [hunk 2]
```diff
@@ -33,3 +33,8 @@ export const install = async () => {
...
+    let return_code = await exec("attic", ["-V"]);
+    return return_code==0;
```
Comment (ryanccn) [inline src/stages/install.ts:39]:
```suggestion
    return return_code === 0;
```

**Review by ryanccn — CHANGES_REQUESTED** [review]
Please use the designated formatter (Prettier)! `pnpm run format` should do the trick.

**Comment by pfzetto** [comment]
Sorry, I implemented the suggested changes and ran the formatter.

**Review by getchoo — APPROVED** [review]
lgtm

**Review by ryanccn — APPROVED** [review]

**Pull request merged**